README.md

```diff
@@ -7,11 +7,23 @@ HC-SR04 is an ultrasonic sensor that measures distances from 2 to 400cm.
 This is a simple library for it!
 
 ## Usage
-Sensor is initialized by creating instance of class UltraSonicDistanceSensor and providing trigger and echo pins.
+Sensor is initialized by creating instance of class UltraSonicDistanceSensor and providing trigger and echo pins.  
+This assumes HC-SR04 and set default value for max distance to 4m.
 ```c
 UltraSonicDistanceSensor sensor(triggerPin, echoPin);
 ```
 
+If another sensor is used or you want to set another distance according to environment in which sensor is used, you can set a maximum distance in cm.
```

Comment:
```suggestion
Default value for maximum measurement distance is 4m, since HC-SR04 sensor can't measure reliably beyond that.
However, if you are using another sensor or if you you don't care about distances larger than some value, you can set a maximum distance in cm yourself.
```